cfme/tests/ansible/test_embedded_ansible_automate.py

```diff
@@ -162,14 +114,16 @@ def alert_profile(appliance, alert, full_template_vm_modscope):
     _alert_profile.delete_if_exists()
 
 
-def test_automate_ansible_playbook_method_type_crud(appliance, ansible_repository, domain,
-        namespace, klass):
+@pytest.mark.meta(automates=[1729999, 1740769])
```

Comment:
**Question:**
I don't think cloned `bugzilla:1740769`  should be a part of `meta marker`, what do you think .?